deploy/charts/monitoring/dashboards/central.json

```diff
@@ -19,25 +19,29 @@
     ]
   },
   "editable": true,
-  "fiscalYearStartMonth": 0,
```

Comment:
The name of this file is no longer accurate, as it is no longer a dashboard just for central. Should it be renamed? If so, what should it be renamed to?

Reply:
How about `stackrox-core-dashboard.json`?

Reply:
Done